### PR TITLE
feat: add all backend settings to the forms

### PR DESCRIPTION
**Pull request by davidemarcoli**
## Summary by CodeRabbit

- **New Features**
	- Added new fields in forms for enhanced user configuration, including options for "Trakt," "Jellyfin," and "Emby" settings.
	- Introduced new checkbox fields for enabling aliases and bucket limits in scrapers.
	- Enhanced media server form with new fields for Jellyfin and Emby services.
	- Added a checkbox for fetching the most watched items in the content form.
	- Streamlined video extensions management in the general form.
	- Updated scrapers form with new fields for cached results and video 3D parameters.

- **Bug Fixes**
	- Improved error handling in forms to maintain user notifications during submissions.

- **Chores**
	- Updated the `.gitignore` file to include additional ignored files and directories for better project management.

### CODE REVIEW

File: src/lib/forms/general-form.svelte [hunk 1]
```diff
@@ -71,6 +79,53 @@
...
+				on:click={() => {
+					addField('video_extensions');
+				}}
+			>
+				<Plus class="h-4 w-4" />
+			</Form.Button>
+		</div>
+	</ArrayField>
+
+	<CheckboxField {form} name="prefer_speed_over_quality" {formData} />
```
Comment (Gaisberg) [inline src/lib/forms/general-form.svelte:127]: This has been removed

File: src/lib/forms/general-form.svelte [hunk 2]
```diff
@@ -109,29 +164,63 @@
...
+				isProtected={true}
+			/>
 		</div>
 
 		<div transition:slide>
 			<CheckboxField
 				{form}
 				name="alldebrid_proxy_enabled"
-				label="All-Debrid Proxy"
+				label="AllDebrid Proxy"
```
Comment (Gaisberg) [inline src/lib/forms/general-form.svelte:187]: Proxies are now not downloader specific

**Review by Gaisberg — CHANGES_REQUESTED** [review]

File: src/lib/forms/general-form.svelte (hunk 1 above)
Comment (davidemarcoli) [reply src/lib/forms/general-form.svelte]: Done, I've removed it

File: src/lib/forms/general-form.svelte (hunk 2 above)
Comment (davidemarcoli) [reply src/lib/forms/general-form.svelte]: Done, I've moved it

**Review by AyushSehrawat — APPROVED** [review]

**Pull request merged**